### PR TITLE
create list user page

### PR DESCRIPTION
**Pull request by tungblvt**
==> Title: create list user page

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/13568

### CODE REVIEW

**Comment by tungblvt** [comment]
ready

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <div class="col-md-12">
+      <h1><%= t "list_users.all_users" %></h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <table class="table-hover table list-users">
+        <tr>
+          <th>ID</th>
+          <th>Name</th>
```
Comment (huuphuc93) [inline app/views/users/index.html.erb]: I18n

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <div class="row">
+    <div class="col-md-12">
+      <h1><%= t "list_users.all_users" %></h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <table class="table-hover table list-users">
+        <tr>
+          <th>ID</th>
```
Comment (huuphuc93) [inline app/views/users/index.html.erb]: để là STT đi em, k để ID

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          <th>Action</th>
+        </tr>
+        <% @users.each do |user| %>
+        <tr>
+          <td>#<%= user.id %></td>
+          <td><%= user.name %></td>
+          <td><%= user.role %></td>
+          <td>
+            <a href="">Edit</a>
+            <a href="">Delete</a>
```
Comment (huuphuc93) [inline app/views/users/index.html.erb]: I18n

**Comment by tungblvt** [comment]
ready

**Pull request merged**